lms/static/js/spec/shoppingcart/shoppingcart_spec.js

```diff
@@ -7,7 +7,10 @@ define(['common/js/spec_helpers/ajax_helpers', 'js/shoppingcart/shoppingcart'],
             var requests = null;
 
             beforeEach(function() {
-                setFixtures('<section class="wrapper confirm-enrollment shopping-cart cart-view"><form action="" method="post"><input type="hidden" name="" value="" /><i class="icon fa fa-caret-right"></i><input type="submit" value="Payment"/></form></section>');
+                setFixtures('<section class="wrapper confirm-enrollment shopping-cart cart-view">' +
```

Comment:
This could also potentially be the cause of the quality error, but given that it is a spec file, I have no desire to do the HTML encoding.